dns/prxu.org-hosted_zone.yml

```diff
@@ -104,47 +99,26 @@ Resources:
             DNSName: dualstack.infra-dovet-1y2ncjszlrdmz-1988316390.us-east-1.elb.amazonaws.com.
             HostedZoneId: Z35SXDOTRQ7X7K
   Mx:
-    Type: "AWS::Route53::RecordSetGroup"
+    Type: AWS::Route53::RecordSetGroup
     Properties:
       Comment: MX Records
       HostedZoneId: !Ref HostedZone
       RecordSets:
         - ResourceRecords:
-            - "1 ASPMX.L.GOOGLE.COM."
-            - "5 ALT1.ASPMX.L.GOOGLE.COM."
-            - "5 ALT2.ASPMX.L.GOOGLE.COM."
-            - "10 ASPMX2.GOOGLEMAIL.COM."
-            - "10 ASPMX3.GOOGLEMAIL.COM."
+            - 1 ASPMX.L.GOOGLE.COM.
+            - 5 ALT1.ASPMX.L.GOOGLE.COM.
+            - 5 ALT2.ASPMX.L.GOOGLE.COM.
+            - 10 ASPMX2.GOOGLEMAIL.COM.
+            - 10 ASPMX3.GOOGLEMAIL.COM.
           TTL: "300"
           Type: MX
           Name: !Ref Domain
-        - ResourceRecords:
-            - "1 ASPMX.L.GOOGLE.COM."
-            - "5 ALT1.ASPMX.L.GOOGLE.COM."
-            - "5 ALT2.ASPMX.L.GOOGLE.COM."
-            - "10 ASPMX2.GOOGLEMAIL.COM."
-            - "10 ASPMX3.GOOGLEMAIL.COM."
-          TTL: "300"
-          Type: MX
-          Name: !Sub tal.dovetail.${Domain}
   CloudFront:
-    Type: "AWS::Route53::RecordSetGroup"
+    Type: AWS::Route53::RecordSetGroup
     Properties:
       Comment: CloudFront
       HostedZoneId: !Ref HostedZone
       RecordSets:
-        - Type: A
-          Name: !Sub userupload.${Domain}
-          AliasTarget:
-            DNSName: dsn15m3yob5tf.cloudfront.net.
-            # Global CloudFront hosted zone ID
-            HostedZoneId: Z2FDTNDATAQYW2
-        - Type: AAAA
-          Name: !Sub userupload.${Domain}
```

Comment:
userupload?  Was this a `prx-up` thing?

Reply:
It was a CloudFront that wasn't connected to anything